web/src/components/molecules/Content/Table/index.tsx

```diff
@@ -196,6 +238,21 @@ const ContentTable: React.FC<Props> = ({
     [t, sort?.field?.type, sort?.direction, selectedItem?.id, onItemSelect],
   );
 
+  const contentColumn: ProColumns<ContentTableField>[] | undefined = useMemo(
+    () =>
+      contentTableColumns?.map(column => ({
```

Comment:
I think it's better that adding property to contentTableColumns in src\components\organisms\Project\Content\ContentList\hooks.ts, but if you did that, is there any problems?

Reply:
No problem on that, I changed it because it would affect the filtering.
If filtering is ok, we can use "ProColumns<ContentTableField>[] | undefined".

Reply:
Filtering is ok! You can add the properties in hooks.ts!

Reply:
I did this to separate data fields and visualization fields.